lib/linguist/languages.yml

```diff
@@ -615,6 +615,18 @@ CSV:
   extensions:
   - ".csv"
   language_id: 51
+Cuneiform:
+  type: programming
+  extensions:
+  - ".cf"
+  - ".cfl"
+  - ".cuneiform"
+  interpreters:
+  - "cuneiform"
+  - "cf_client"
+  color: "#2772b0"
+  tm_scope: none
```

Comment:
Even though it's a different language, you can use Ruby's grammar for Cuneiform highlighting. The results [look decent enough](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=source.ruby&grammar_format=auto&grammar_url=&grammar_text=&code_source=from-text&code_url=&code=def+inc%28+x+%3A+Str+%29+-%3E+Str+%7B%0D%0A%0D%0A++def+inc%28+x+%3A+Str+%29+-%3E+%3Cy+%3A+Str%3E+in+Python+*%7B%0D%0A++++y+%3D+int%28+x+%29%2B1%0D%0A++%7D*%0D%0A%0D%0A++%28+inc%28+x+%3D+x+%29%7Cy+%29%0D%0A%7D%0D%0A%0D%0Adef+dec%28+x+%3A+Str+%29+-%3E+Str+%7B%0D%0A%0D%0A++def+dec%28+x+%3A+Str+%29+-%3E+%3Cy+%3A+Str%3E+in+Python+*%7B%0D%0A++++y+%3D+int%28+x+%29-1%0D%0A++%7D*%0D%0A%0D%0A++%28+dec%28+x+%3D+x+%29%7Cy+%29%0D%0A%7D).

If that isn't good enough, just give me the formal language specification, and I'll write a grammar for you. 😉